Automattic/wp-calypso
# Seller Experience: Added store name and tagline view.

#### Changes proposed in this Pull Request

* Added new view so the user can enter the store name and a tagline - under the `seller-experience` feature flag.

#### Testing instructions



* Access `start/setup-site/intent?siteSlug=<SITE SLUG>` (which doesn't enable the `seller-experience` feature flag)
* Things should be unchanged!
![image](https://user-images.githubusercontent.com/3801502/150015621-5a83ad9d-105e-4122-a6f6-40dfbc841898.png)


* Access `start/setup-site/intent?siteSlug=<SITE SLUG>&flags=seller-experience` (which enables the `seller-experience` feature flag)
* Click the "Start Selling" button
![image](https://user-images.githubusercontent.com/3801502/150015569-2747a047-f39c-4373-b577-f9d4d95a1402.png)
* Check the new view

![image](https://user-images.githubusercontent.com/3801502/150198467-685f83bd-629e-4cb6-9b3f-8fd13dfa31bb.png)




Related to #60154
